infrastructure/cdn-in-a-box/dns/Dockerfile

```diff
@@ -23,10 +23,10 @@ ENV BIND_USER=bind \
 RUN echo 'APT::Install-Recommends 0;' >> /etc/apt/apt.conf.d/01norecommends \
  && echo 'APT::Install-Suggests 0;' >> /etc/apt/apt.conf.d/01norecommends \
  && apt-get update \
- && DEBIAN_FRONTEND=noninteractive apt-get install -y vim.tiny wget net-tools sudo net-tools ca-certificates unzip apt-transport-https \
+ && DEBIAN_FRONTEND=noninteractive apt-get --no-install-recommends install -y vim.tiny wget net-tools sudo net-tools ca-certificates unzip apt-transport-https \
```

Comment:
The lines above

```dockerfile
RUN echo 'APT::Install-Recommends 0;' >> /etc/apt/apt.conf.d/01norecommends \
 && echo 'APT::Install-Suggests 0;' >> /etc/apt/apt.conf.d/01norecommends \
```
updating the APT config to skip package recommendations, so there is no need to pass the `--no-install-recommends` option with every apt or apt-get command.